scripts/entrypoint.sh

```diff
@@ -58,6 +58,18 @@ wait_for_ipam() {
     return 1
 }
 
+echo -n "Copying portmap binary... "
+
+if [[ -f "$HOST_CNI_BIN_PATH" ]]
+then
+    mv "$HOST_CNI_BIN_PATH" "$HOST_CNI_BIN_PATH.old"
```

Comment:
This will rename the whole bin directory. How about something like
```bash
HOST_PORTMAP="$HOST_CNI_BIN_PATH/portmap"
if [[ -f "$HOST_PORTMAP" ]]
then
    mv "$HOST_PORTMAP" "$HOST_PORTMAP.old"
```

Reply:
Ah yes, of course. Thanks